include/trgdataformats/TriggerCandidateData.hpp

```diff
@@ -45,6 +45,7 @@ struct TriggerCandidateData
     kCTBCosmicJura = 21,
     kCTBCosmicSaleve = 22,
     kNeutronSourceCalib = 23,
+    kChannelAdjacency = 24,
```

Comment:
I'm wondering whether it would be worth moving `kChannelAdjcency` to be somewhere before all the `CTB`s, but I'm not sure if I care enough here. @MRiganSUSX ?

Reply:
It is essentially move it now or never move it, but then the future would want consistency if we move it now. It may have been better for the `CTB`s to be all at the top. I'm in favor of leaving it here.